### PR TITLE
Updated the issues faced in networking

### PR DESCRIPTION
**Pull request by manish-pra**

### CODE REVIEW

**Comment by AndreaCensi** [comment]
@manish-pra test fails

**Comment by manish-pra** [comment]
@AndreaCensi Can you try to merge now? I changed the error line

File: book/opmanual_duckiebot/atoms_18_setup_troubleshooting/20_network_troubleshooting.md [hunk 1]
```diff
@@ -57,3 +71,18 @@ Restart Avahi by running the command
 
 ## I can SSH to the Duckiebot but not without a password {status=draft}
 
+Check the file ~.ssh/config, make sure you add your ssh key there, incase it doesn't exists.
+
+The `init_sd_card` should generate a para in the above file in the following format:
+
+# --- init_sd_card generated ---
```
Comment (AndreaCensi) [inline book/opmanual_duckiebot/atoms_18_setup_troubleshooting/20_network_troubleshooting.md]: Need to quote this

File: book/opmanual_duckiebot/atoms_18_setup_troubleshooting/20_network_troubleshooting.md [hunk 2]
```diff
@@ -7,7 +7,21 @@ All errors that prevent the Duckiebot to answer to pings and SSH.
 
 ## I cannot ping the Duckiebot {status=draft}
 
+Sympton: ping ![robot_name] does not work.
```
Comment (AndreaCensi) [inline book/opmanual_duckiebot/atoms_18_setup_troubleshooting/20_network_troubleshooting.md]: need to quote ![robot_name]

File: book/opmanual_duckiebot/atoms_18_setup_troubleshooting/20_network_troubleshooting.md [hunk 3]
```diff
@@ -7,7 +7,21 @@ All errors that prevent the Duckiebot to answer to pings and SSH.
...
 
+IMP: Check if you and duciebot intent to connect to the same network.
+
+You can derive conclusions by following next few steps, and decide based upon it:
+
+Step 1: Check that your rpi is responsive. Check for the led blinking on the rpi.
```
Comment (AndreaCensi) [inline book/opmanual_duckiebot/atoms_18_setup_troubleshooting/20_network_troubleshooting.md]: Don't abbreviate rpi.

**Comment by manish-pra** [comment]
@AndreaCensi: Thanks for the suggestions. I have incorporated them. Please check and merge now.

**Pull request merged**